data-raw/rs_supprs_onco_imwg.R

```diff
@@ -0,0 +1,123 @@
+# Packages: ---------------------------------------------------------------
+library(dplyr)
+
+# Tribble data: -----------------------------------------------------------
```

Comment:
Please could you shorten section labels to four dashes (package convention)

Reply:
Updated